vars/piperExecuteBin.groovy

```diff
@@ -156,6 +156,10 @@ void dockerWrapper(script, stepName, config, body) {
 
 // reused in sonarExecuteScan
 void credentialWrapper(config, List credentialInfo, body) {
+    if (config.containsKey('vaultAppRoleCredentialId') && config.containsKey('vaultAppRoleSecretCredentialId')) {
+        credentialInfo = [[type: 'token', id: 'vaultAppRoleCredentialId', env: ['PIPER_vaultRoleID']],
```

Comment:
`vaultAppRole` and `vaultRole ` looks like a mismatch..

Reply:
true